ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs

```diff
@@ -59,7 +59,19 @@
                 {{#if hostCatalog.isStatic}}
                   <Rose::Badge>
                     {{t
-                      (concat 'resources.host-catalog.types.' hostCatalog.type)
+                      (concat
+                        'resources.host-catalog.types.'
+                        hostCatalog.compositeType
+                      )
+                    }}
+                  </Rose::Badge>
+                {{else if hostCatalog.isUnknown}}
+                  <Rose::Badge>
+                    {{t
+                      (concat
```

Comment:
I will copy this approach for host-set and host once this get merged (I need the changes you add on `resources/en-us.yaml`